### DISCUSSION TITLE
RTSP video streaming?

### DISCUSSION THREAD

**Discussion by pepeRossRobotics** [foxglove/community#18] [Q&A]
Hello,
I was wondering if it is possible to stream RTSP videos using Foxglove. 
The reason why this is useful for us is that it is a very lo CPU intensive task an there are plenty of cameras that do it. So it is great for tele operation and low bandwidth video streaming.

**Comment by jtbandes** [comment]
This is not currently possible. Please see https://github.com/foxglove/studio/issues/2064 for more discussion regarding video encodings.
